src/sass/layout/global.scss

```diff
@@ -0,0 +1,31 @@
+
+// font-family: 'Lato', sans-serif;
+// font-family: 'Roboto Slab', serif;
+
+$font-First: 'Lato', sans-serif;
+$font-Second: 'Roboto Slab', serif;
+
+// ###########
+// ###Color###
+// ###########
+$color-first: #f7f7f7;
```

Comment:
tutaj nazwalbym to odpowiednio:

- color-primary
- color-secondary

albo odpowiednia tak jakie sa kolory, np:
- color-grey
- color-light-blue
- color-dark-red